FirebasePerformance/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Version 8.1.1
```

Comment:
Patch updates are only for out of band critical fixes. This should be 8.2.0

Reply:
Thanks! Updated!